pkg/importer/spanner/import

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env sh
+arrai r import.arrai $1 > $2
```

Comment:
Nit: this file doesn't appear to add any particular value, and worst case could encourage non-standard approaches to running arrai scripts. Is there some context in which simply writing `arrai run import foo > bar` is too inconvenient?

Feel free to have scripts like this locally, but don't include in the design of a feature or commit to Git.

Reply:
It wouldn't be there in the final PR probably, only serves as the convenience script currently.

Reply:
Also the fact that this would be integrated to sysl command line eventually, this script would just serve as a test ground until we actually integrate with sysl command line.